components/root/root.test.jsx

```diff
@@ -167,4 +191,11 @@ describe('components/Root', () => {
         wrapper.setProps(props2);
         expect(props.history.push).toHaveBeenLastCalledWith('/signup_user_complete');
     });
+
+    test('should call for enableRudderEvents on call of onConfigLoaded if url and key for rudder is set', () => {
```

Comment:
Can we verify the inverse as well, such that we don't enable events when the url or rudder key is not et?

Reply:
Did